web_dashboard_tile/models/tile_tile.py

```diff
@@ -0,0 +1,276 @@
+
+    def _get_eval_context(self):
+        def _context_today():
+            return fields.Date.from_string(fields.Date.context_today(self))
+        context = self.env.context.copy()
+        context.update({
+            'time': time,
+            'datetime': datetime,
+            'relativedelta': relativedelta,
+            'context_today': _context_today,
+            'current_date': fields.Date.today(),
+        })
+        return context
+
+    # Column Section
+    name = fields.Char(required=True)
+    sequence = fields.Integer(default=0, required=True)
+    user_id = fields.Many2one('res.users', 'User')
+    background_color = fields.Char(default='#0E6C7E', oldname='color')
+    font_color = fields.Char(default='#FFFFFF')
+
+    group_ids = fields.Many2many(
+        'res.groups',
+        string='Groups',
+        help='If this field is set, only users of this group can view this '
+             'tile. Please note that it will only work for global tiles '
+             '(that is, when User field is left empty)')
+
+    model_id = fields.Many2one('ir.model', 'Model', required=True)
+    domain = fields.Text(default='[]')
+    action_id = fields.Many2one('ir.actions.act_window', 'Action')
+
+    active = fields.Boolean(
+        compute='_compute_active',
+        search='_search_active',
+        readonly=True)
+
+    # Primary Value
+    primary_function = fields.Selection(
+        FIELD_FUNCTION_SELECTION,
+        string='Function',
+        default='count')
+    primary_field_id = fields.Many2one(
+        'ir.model.fields',
+        string='Field',
+        domain="[('model_id', '=', model_id),"
+               " ('ttype', 'in', ['float', 'integer'])]")
+    primary_format = fields.Char(
+        string='Format',
+        help='Python Format String valid with str.format()\n'
+             'ie: \'{:,} Kgs\' will output \'1,000 Kgs\' if value is 1000.')
+    primary_value = fields.Char(
+        string='Value',
+        compute='_compute_data')
+    primary_helper = fields.Char(
+        string='Helper',
+        compute='_compute_helper')
+
+    # Secondary Value
+    secondary_function = fields.Selection(
+        FIELD_FUNCTION_SELECTION,
+        string='Secondary Function')
+    secondary_field_id = fields.Many2one(
+        'ir.model.fields',
+        string='Secondary Field',
+        domain="[('model_id', '=', model_id),"
+               " ('ttype', 'in', ['float', 'integer'])]")
+    secondary_format = fields.Char(
+        string='Secondary Format',
+        help='Python Format String valid with str.format()\n'
+             'ie: \'{:,} Kgs\' will output \'1,000 Kgs\' if value is 1000.')
+    secondary_value = fields.Char(
+        string='Secondary Value',
+        compute='_compute_data')
+    secondary_helper = fields.Char(
+        string='Secondary Helper',
+        compute='_compute_helper')
+
+    error = fields.Char(
+        string='Error Details',
+        compute='_compute_data')
+
+    @api.one
```

Comment:
Remove api.one from everywhere and replace with api.multi and deal with the fact that `self` might contain more than one records.